### PR TITLE
336 change links in menus

### LINKED ISSUE #336: Change links in menus
- in left menu: change Support to "Forum support" (name only changes)
- right buttom menu
  - link to edit page
  - link support to FAQ

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

**Review by louilinn — COMMENTED** [review]
See comments in https://github.com/CirclesUBI/circles-myxogastria/pull/374#pullrequestreview-999806623

File: src/routes.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export const SETTINGS_PATH = '/settings';
 export const SHARE_PATH = '/share';
 export const VALIDATION_PATH = '/validation';
 export const WELCOME_PATH = '/welcome';
+export const EDIT_PROFILE = '/editprofile';
```
Comment (louilinn) [inline src/routes.js:51]: When working on the edit profile functionality the route is simply /edit. I think it is better that the route is introduced when we have the functionality

Comment (mikozet) [reply src/routes.js:51]: Should I maybe remove `edit profile` from menu list?  Or change it for 'edit'

Comment (louilinn) [reply src/routes.js:51]: It is ok :) I will change it later instead if I want /edit rather than editprofile

**Pull request merged**